tracer/test/Datadog.Trace.Tests/Agent/MessagePack/SpanMessagePackFormatterTests.cs

```diff
@@ -208,6 +208,230 @@ public void SpanLink_Tag_Serialization()
         }
     }
 
+    [Theory]
+    [InlineData("true")]
+    [InlineData("false")]
+    [InlineData("null")]
+    public void SpanEvent_Tag_Serialization(string nativeSpanEventsEnabled)
```

Comment:
Can you change this to a nullable boolean? I think that will better represent the three states of "not set", "false", and "true".

```suggestion
    public void SpanEvent_Tag_Serialization(bool? nativeSpanEventsEnabled)
```